src/api/nodeRpcCall.js

```diff
@@ -1695,6 +1695,12 @@ const stakingBondExtra = async (value, walletAddress, callback) => {
   });
 }
 
+const getPalletIds = async () => {
+  const api = await getApi();
+  const pallets = Object.keys(api.consts).map(palletName => ({palletName, palletId: api.consts[palletName].palletId}))
```

Comment:
I don't really care, it's good as it is now, just adding this in case you're not aware of `Object.entries`.
> The Object.entries() static method returns an array of a given object's own enumerable string-keyed property key-value pairs.

```suggestion
  const pallets = Object.entries(api.consts).map(([palletName, palletConsts]) => ({palletName, palletId: palletConsts.palletId}))
```